src/tasks.py

```diff
@@ -286,10 +297,44 @@ def get_depth_by_time_at_point(model_id: int, lat: float, lng: float) -> DepthTi
     return DepthTimePlot(depths, times)
 
 
+@app.task(base=OnFailureStateTask)
+def retrieve_medusa_input_parameters(scenario_id: int) -> Dict[str, Union[str, float]]:
+    """
+    Retrieve input parameters for the current scenario id. This is used for app file
+
+    Parameters
+    ----------
+    scenario_id: int
+        The scenario ID of the pollution model run
+
+    Returns
+    -------
+    MedusaRainfallEventGeometry: class
+        Rainfall event parameters for MEDUSA 2.0 model with geometry
+    """
+    # Get rainfall information
+    medusa_rainfall_event = retrieve_input_parameters(scenario_id)
+
+    if medusa_rainfall_event != None:
+        # Write rainfall information into a dictionary
+        medusa_rainfall_dictionary = {
+            "antecedent_dry_days": float(medusa_rainfall_event.antecedent_dry_days),
+            "average_rain_intensity": float(medusa_rainfall_event.average_rain_intensity),
+            "event_duration": float(medusa_rainfall_event.event_duration),
+            "rainfall_ph": float(medusa_rainfall_event.rainfall_ph),
+            "geometry": str(medusa_rainfall_event.geometry)
+        }
+        return medusa_rainfall_dictionary
+
+    else:
+        return None
```

Comment:
```suggestion
    if medusa_rainfall_event == None:
        # The specified scenario_id does not exist
        return None
    # Write rainfall information into a dictionary
    medusa_rainfall_dictionary = {
        "antecedent_dry_days": medusa_rainfall_event.antecedent_dry_days,
        "average_rain_intensity": medusa_rainfall_event.average_rain_intensity,
        "event_duration": medusa_rainfall_event.event_duration,
        "rainfall_ph": medusa_rainfall_event.rainfall_ph,
        "geometry": medusa_rainfall_event.geometry
    }
    return medusa_rainfall_dictionary
```

Reply:
These changes simplify the code flow (inverting the if) and also remove the casts that aren't necessary